src/main/java/audio/Sonifier.java

```diff
@@ -48,6 +47,13 @@ public static PlaybackController sonify(InstrumentDataRaw[] instrumentDataRaw, E
         double[][] outArrays = concatArrays(evInstrs, synthLines);
 
         double[] out = Mixer.mixAudioStreams(outArrays, evInstrs.length, synthLines.length - 1);
+
+        GlobalFxParam globalFxParam = new Harmonizer(globalFxParamRaw, lengthInBeats).gHarmonize();
+        if(globalFxParam.getDelayReverb() != null && globalFxParam.getDelayReverb() != null)
```

Comment:
I don't think you meant to check the same condition twice with this `and`-statement, did you?